Javascript/jq-test.js

```diff
@@ -1,69 +1,17 @@
-$( document ).ready(function() {
-    console.log( "ready!" );
-
+$(function() {
     $('#settings').click(function(){
       $('.bkg-menu').removeClass("hide");
+      $('#set-title').addClass("hide");
     });
 
     $('#profile').click(function(){
       $('.bkg-menu').removeClass("hide");
+      $('#pro-title').addClass("hide");
     });
 
     $('#menu-close').click(function(){
       $('.bkg-menu').addClass("hide");
-    });
-
-    $(function() {
-
-        $('#settings').click(function() {
-
-            if ($('#set-title').hasClass("hide")) {
-                $('#set-title').removeClass('hide');
-            }
-            else if (!$('#set-title').hasClass("hide")) {
```

Comment:
`#settings` cannot be clicked when `#set-title` is visible, so this is and the if are redundant. Also `else if (!$('#set-title').hasClass("hide")) {` would be the same as `else {`, if this was needed, as if `$('#set-title').hasClass("hide")` is false, `!$('#set-title').hasClass("hide")` is always true